Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -175,18 +176,34 @@ class CredentialsManagerSpec: QuickSpec {
 
             context("renew") {
 
-                it("should yield new credentials") {
+                it("should yield new credentials, maintain refresh token") {
                     credentials = Credentials(accessToken: AccessToken, tokenType: TokenType, idToken: IdToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: -3600))
                     _ = credentialsManager.store(credentials: credentials)
                     waitUntil(timeout: 2) { done in
                         credentialsManager.credentials { error = $0; newCredentials = $1
                             expect(error).to(beNil())
-                            expect(newCredentials?.accessToken) == AccessToken
+                            expect(newCredentials?.accessToken) == NewAccessToken
+                            expect(newCredentials?.refreshToken) == RefreshToken
                             done()
                         }
                     }
                 }
 
+                it("should store new credentials") {
+                    credentials = Credentials(accessToken: AccessToken, tokenType: TokenType, idToken: IdToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: -3600))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials { error = $0; newCredentials = $1
+                            expect(error).to(beNil())
+                            credentialsManager.credentials {
+                                expect($1!.accessToken) == NewAccessToken
```

Comment:
here you use `$1!` but in the above test you refer to the same value as `newCredentials`. Why?

Reply:
It's not the same value, the second `credentialsManager.credentials {` is another closure so the $0,$1 are different from the first closure.